man1/aur-rpc.1

```diff
@@ -12,6 +13,12 @@ aur-rpc \- send GET requests to aurweb
 
 .SH OPTIONS
 
+.SH ENVIRONMENT
+.B AURURI
+.RS
+Sets the default URI when fetching packages.
```

Comment:
`aur-rpc` queries AurJson, it does not fetch packages. i.e. "default URI for the aurweb RPC interface" is what is used here.